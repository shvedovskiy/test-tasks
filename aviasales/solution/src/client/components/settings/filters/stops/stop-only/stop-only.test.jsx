/* eslint-disable no-undef */
import React from 'react';

import StopOnly from './stop-only';


describe('<StopOnly />', () => {
  const props = {
    handleSelect: jest.fn(),
  };
  const stopOnly = shallow(<StopOnly {...props} />);

  it('renders properly', () => {
    expect(stopOnly).toMatchSnapshot();
  });
});
