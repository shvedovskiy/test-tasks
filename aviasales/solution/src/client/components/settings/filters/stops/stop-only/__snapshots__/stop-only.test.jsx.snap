// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StopOnly /> renders properly 1`] = `
<styled__StopOnlyButton>
  <styled__Button
    onClick={[MockFunction]}
    onKeyDown={[Function]}
  >
    Только
  </styled__Button>
</styled__StopOnlyButton>
`;
